src/spid-callbacks.js

```diff
@@ -0,0 +1,32 @@
+/*global module:false*/
+var _callbacks = {};
+
+function ensureArray(key) {
+    if(!_callbacks[key]) {
```

Comment:
Not a hard requirement but this might be more "proper" but then it probably should go to `spid-util.js`:

```javascript
function _isArray(arg) {
    // Code from MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray
    return Object.prototype.toString.call(arg) === '[object Array]';
}
```

Reply:
👍

Reply:
Is there a need to polyfill that even?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray
Support since ie9.